frontend/app/monitoring/MemberMetrics.scala

```diff
@@ -26,6 +28,20 @@ class MemberMetrics(val backendEnv: String) extends TouchpointBackendMetrics {
     put(s"failed-sign-up-${tier.name}")
   }
 
+  def putCreationOfPaidSubscription(paymentMethod: Option[PaymentMethod]) = {
+
+    val methodName = paymentMethod match {
+      case Some(method) => method.getClass.getSimpleName
+      case None => "UnknownPaymentMethod"
```

Comment:
Sigh. `paymentMethod` will probably never be `None` for a paid subscription - but will be `None` when a Friend signs up. `createPaymentMethod()` should possibly return just a `PaymentMethod`, not an `Option[PaymentMethod]`, as it doesn't seem to be used in the Free flow?

https://github.com/guardian/membership-frontend/blob/13cf75ae3e7/frontend/app/services/MemberService.scala#L459

Anyway, not a blocker for this change, but a bit sad...